### ISSUE TITLE
ModuleNotFoundError: No module named 'findopp'

### ISSUE THREAD

**Issue by texadactyl** [oreste-t/turboSETI#1]
pip3 uninstall turbo_seti
Downloaded the ZIP file.
Unzipped.
cd turboSETI-master
python3  setup.py  install

So far, so good.  But:

turboSETI  $DATA/voyager/Voyager1.single_coarse.fine_res.h5
```
Traceback (most recent call last):
  File "/usr/local/bin/turboSETI", line 11, in <module>
    load_entry_point('turbo-seti==1.0.2', 'console_scripts', 'turboSETI')()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 490, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2854, in load_entry_point
    return ep.load()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2445, in load
    return self.resolve()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2451, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python3.8/dist-packages/turbo_seti-1.0.2-py3.8-linux-x86_64.egg/turbo_seti/__init__.py", line 1, in <module>
    from .findoppler import seti_event
  File "/usr/local/lib/python3.8/dist-packages/turbo_seti-1.0.2-py3.8-linux-x86_64.egg/turbo_seti/findoppler/seti_event.py", line 3, in <module>
    from findopp import FinDoppler
ModuleNotFoundError: No module named 'findopp'
```

**Comment by texadactyl** [comment]
Same result when I use a Python3 user program to import FinDoppler:
```
"""
Turbo Seti FinDoppler Test
"""
import time
from os import getenv
from turbo_seti.findoppler.findopp import FinDoppler

HOME = getenv("HOME")
DATADIR = HOME + "/data/voyager/"
DATAFILE = DATADIR + "Voyager1.single_coarse.fine_res.h5"

print("Calling FinDoppler({})".format(DATAFILE))
print("Output directory: {}".format(DATADIR))
start_time = time.time()
FD = FinDoppler(DATAFILE, 2, out_dir=DATADIR)
FD.search()
stop_time = time.time()
et = stop_time - start_time
print("Elapsed time = {} seconds".format(et))

```
This Python program runs without issues with UCBerkeleySETI/turbo_seti.

**Comment by oreste-t** [comment]
I replicated the bug. This is occurring because the following imports:

```from helper_functions import chan_freq```
 on line 5 of file_writers.py

```
from data_handler import DATAHandle
from file_writers import FileWriter, LogWriter
from helper_functions import *
```
on lines 11-13 of findopp.py

```
from findopp import FinDoppler
```
on line 3 on seti_event.py

should be replaced with the following relative imports respectively:

```from .helper_functions import chan_freq```
 on line 5 of file_writers.py

```
from .data_handler import DATAHandle
from .file_writers import FileWriter, LogWriter
from .helper_functions import *
```
on lines 11-13 of findopp.py

```
from .findopp import FinDoppler
```
on line 3 on seti_event.py

I changed this because my debugger is a bit wonky and errors unless I remove the "." for some relative import statements. I changed them back and retried your steps, and everything worked this time. If you would like to try out this code, feel free to make those changes to your local version, but I'm keeping the repo as is currently since it is more convenient for my debugging purposes. This isn't intended to be a production version of turboSETI, it's more like a playground for me to figure out how the current code and my modifications work. All my official changes are made on this fork:
https://github.com/oreste-t/turbo_seti
which is a bit cleaner and has the correct import statements! (Though it doesn't have all my modifications since I'm still experimenting on them in this repo)
